src/foam/nanos/column_config/ColumnConfigPropertyValue.js

```diff
@@ -0,0 +1,370 @@
+/**
+ * @license
+ * Copyright 2020 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+foam.CLASS({
+  package: 'foam.nanos.column',
+  name: 'ColumnConfigToPropertyConverter',
+  documentation: `ColumnConfigToPropertyConverter gathers methods for converting (or mapping) property name to property if required 
+      and returns property and obj, which will return the proprty value on f function call`,
+  javaImports: [
+    'foam.core.ClassInfo',
+    'foam.core.FObject',
+    'foam.core.PropertyInfo',
+    'foam.core.X',
+    'foam.mlang.Expr',
+    'foam.nanos.logger.Logger',
+    'java.lang.reflect.Method'
+  ],
+  methods: [
+    {
+      name: 'filterExportedProps',
+      documentation: 'returns propNames that are properties of the class and are networkTransient',
+      code: function(of, propNames) {
+        var props = of.getAxiomsByClass(foam.core.Property);
+        var allColumnNames = props.map(p => p.name);
+        if ( ! propNames ) {
+          return props.filter(p => p.networkTransient).map(p => p.name);
+        } props = [];
+        return propNames.filter(n => { 
+          return allColumnNames.includes(n.split('.')[0]);// && self.returnProperty(of, n).networkTransient;
```

Comment:
networkTransient is commented out as now a lot of properties are networkTransient

Reply:
should we review networkTransient of some of the columns? or should I just uncomment that?

Reply:
should we set networkTransient default value to true maybe?

Reply:
I would include 'networkTransient' so that these columns are ignored.